examples/bri-3/src/bri/identity/bpiAccounts/agents/mockBpiAccountStorage.agent.ts

```diff
@@ -20,7 +20,13 @@ export class MockBpiAccountsStorageAgent {
   }
 
   async createNewBpiAccount(bpiAccount: BpiAccount): Promise<BpiAccount> {
-    const createdBp = new BpiAccount(v4(), bpiAccount.ownerBpiSubjectAccounts);
+    const createdBp = new BpiAccount(
+      v4(),
+      bpiAccount.ownerBpiSubjectAccounts,
+      bpiAccount.authorizationCondition,
```

Comment:
defining optional will avoid these explicit assignments

Reply:
See above reply, required in the stanard.